docs/grizli/install.rst

```diff
@@ -37,12 +34,139 @@ Set up a ``conda`` environment
+
+.. code-block:: bash
+
+  pip install grizli["jwst"]
+
+or 
+
+.. code-block:: bash
+
+  pip install grizli["jwst","test"]
+
+To minimize conflict of dependencies, install only the ones that you need. 
+
+Additional dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+``pip`` will install all needed dependencies with the exception of the following which
+which need to be installed separately:
+
+.. code-block:: bash
+  
+  pip install git+https://github.com/gbrammer/tristars
+  pip install pyregion
+  
+If you will be working with HST data, you will also need the following two 
+libraries:
+
+.. code-block:: bash
+
+  conda install hstcal
+  pip install git+https://github.com/gbrammer/reprocess_wfc3.git
+
+``lacosmicx``
+#############
+
+``lacosmicx`` is not required for running ``grizli``. If it is not installed, there will
+be no additional cosmic ray rejection performed. ``lacosmicx`` has been superseeded by 
+`astroscrappy <https://github.com/astropy/astroscrappy>`__
+`lacosmicx <https://github.com/cmccully/lacosmicx>`__ is a fast Python implementation of
+Pieter van Dokkum's `L.A.Cosmic <http://www.astro.yale.edu/dokkum/lacosmic/>`__
+(`2001PASP..113.1420V <http://adsabs.harvard.edu/abs/2001PASP..113.1420V>`__) software
+for identifying cosmic rays in single images. The image preparation wrapper scripts in
+`grizli.prep` run ``lacosmicx`` if a supplied list of direct or grism images contains
+only a single file. 
+
+- Change directories to the location where the ``grizli`` repo was cloned before:
+
+.. code-block:: bash
+
+    cd /usr/local/share/python # location from before
+
+- Activate the ``conda`` environment:
+
+.. code-block:: bash
+
+    conda activate grizli39 # or whatever was chosen before
+
+- Fetch the ``lacosmicx`` repo, and change into its directory:
+
+.. code-block:: bash
+
+    git clone https://github.com/cmccully/lacosmicx.git
+    cd lacosmicx
+
+- Install the package into the current environment:
+
+.. code-block:: bash
+
+    python setup.py install
+
+.. note::
+    The `lacosmicx` dependency was removed from `environment.yml` file
+    2019.12.31 because it was breaking on OSX Mojave 10.14.6 with a
+    compilation error like `unsupported option '-fopenmp'`. The workaround
+    below with the Homebrew version of `gcc` may work after verifying the
+    correct path to the `gcc-8` executable:
+    
+    .. code-block:: bash
+
+        brew install gcc
+        CC=/usr/local/Cellar/gcc/10.2.0/bin/gcc-10 pip install git+https://github.com/cmccully/lacosmicx.git
+        
+``eazy-py``
+###########
+
+If you are planning to run simultaneous fits to grism spectra plus photometry using the
+`eazy-py <https://github.com/gbrammer/eazy-py>`_ connection, install ``eazy-py`` from
+the repository to ensure that you get *its* dependencies.
+
+- Change directories to the location where the ``grizli`` repo was cloned before:
+
+.. code-block:: bash
+
+    cd /usr/local/share/python # location from before
```

Comment:
Again, why this location?  It's a bad example location.

Reply:
Thanks! Edited as well as I could.